### ISSUE TITLE
More a Q then an issue

### ISSUE THREAD

**Issue by MichelNivard** [zhenin/HDL#5]
Hi,

absolutely amazing software thanks, I am implementing HDL as an option for h2 and cov_g estimation in the context of GenomicSEM. In your code you estimate h2 twice. First you estimate h1_2, then later you select the eigen values and estimate h11. 

I found that for h1_2 the LL function generally finds a lower minimum (likely because you set the starting value per chunk at the wls estimate, so its close to the optimum, less likely to get stuck in a local minimum than when using a global starting value?).

Maybe it makes sense to use h1_2? or at least implement starting values per chunk?

Best,
Michel

**Comment by zhenin** [comment]
Hi Michel,

Thanks for your interest and comment! Yes, for h11 (the genome-wide SNP-based heritability of trait 1), we estimate it with the following steps:

1. For each chunk, we estimate the heritability carried by the SNPs in the chunk;
2. h1_2 is computed by summing up all chunk heritabilities;
3. We use h1_2 as the initial value to estimate h11.

Using h1_2 makes sense indeed because actually it allows each chunk to have its own h11 parameter. So instead of having only one genome-wide parameter, you have many more parameters, which should give you a larger likelihood.  

Although using h1_2=using more parameters=more flexible, we did not seriously test its robustness. In our paper, we used the same linear mixed model as LDSC did to make the comparison more comprehensive. Next, we will try to develop stratified-HDL, where we will evaluate the robustness of HDL when multiple parameters are introduced.

In summary, theoretically, it makes sense to use h1_2 for extra flexibility. But we have not systematically tested how robust it is.

Best,
Zheng

**Comment by MichelNivard** [comment]
Thats a very clear awnnser, thanks, I'll implement both as options in GenomicSEM.
